ui/app/components/pages/add-token/add-token.component.js

```diff
@@ -106,7 +106,7 @@ class AddToken extends Component {
   }
 
   hasSelected () {
-    const { customAddress = '', selectedTokens = {} } = this.state
```

Comment:
I think we need `selectedTokens = {}` to ensure `Object.keys(selectedTokens)` doesn't error on line 110